content/docs/releases/feature_lifecycle.md

```diff
@@ -88,4 +88,19 @@ The following sections list the features supported in OSM and their versioning s
 | Progressive delivery with Flagger | Alpha |
 | Dapr integration                  | Alpha |
 
-<br/>
\ No newline at end of file
+<br/>
+
+## Helm chart values deprecation
+
+Values in the Helm chart may be deprecated in preparation to be removed in a future release. A value that is deprecated will be prefixed with the `DEPRECATED` keyword alongside its comment in the values file and in the title of the corresponding property in the JSON schema for the Helm values.
```

Comment:
> and in the title of the corresponding property in the JSON schema for the Helm values

This may not be worth mentioning here since I wouldn't expect any user to be looking at the schema directly. I have no issue if we actually note values as deprecated in the schema though.

Reply:
I mentioned this here just as to document that it needs to be in both places. Applies more to developers than end users though.